pyglmnet/pyglmnet.py

```diff
@@ -5,6 +5,7 @@
 
 import numpy as np
 from scipy.special import expit
+from sklearn.base import BaseEstimator, RegressorMixin
```

Comment:
if `score` method is all what you need, maybe we don't even need `BaseEstimator` for now. We can simply add the `score` method and see if it works ...